aptos-move/e2e-move-tests/src/tests/aggregator.rs

```diff
@@ -6,160 +6,172 @@ use crate::{
         add, add_and_materialize, check, destroy, initialize, materialize, materialize_and_add,
         materialize_and_sub, new, sub, sub_add, sub_and_materialize,
     },
-    assert_abort, assert_abort_ref, assert_success,
     tests::common,
-    MoveHarness,
+    BlockSplit, MoveHarness,
 };
 use aptos_language_e2e_tests::account::Account;
-use aptos_types::transaction::SignedTransaction;
+use proptest::prelude::*;
+use test_case::test_case;
 
 fn setup() -> (MoveHarness, Account) {
     initialize(common::test_dir_path("aggregator.data/pack"))
 }
 
-#[test]
-fn test_aggregators_e2e() {
+#[test_case(BlockSplit::Whole)]
+#[test_case(BlockSplit::SingleTxnPerBlock)]
+fn test_aggregators_e2e(block_split: BlockSplit) {
```

Comment:
Why is this test not part of `proptest!` like other tests?

Reply:
proptest is good when you want to "search the subset of the combinatorial space". I.e. it tries smartly to pick and choose which tests to run, and how to find the simplest test case that fails. But you never know if it will run any individual particular set of parameters.

When you want to do exhaustive tests of the combinatorial space - test_case is better.